site/ko/beta/tutorials/estimators/linear.ipynb

```diff
@@ -0,0 +1,679 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "y86q1fj44lZs"
+   },
+   "outputs": [],
+   "source": [
+    "dftrain.describe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "8JSa_duD4tFZ"
+   },
+   "source": [
+    "훈련셋은 627개의 사례로 평가셋은 264개의 사례로 구성되어 있습니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "Fs3Nu5pV4v5J"
+   },
+   "outputs": [],
+   "source": [
+    "dftrain.shape[0], dfeval.shape[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "RxCA4Nr45AfF"
+   },
+   "source": [
+    "대부분의 승객은 20대와 30대 입니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "RYeCMm7K40ZN"
+   },
+   "outputs": [],
+   "source": [
+    "dftrain.age.hist(bins=20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "DItSwJ_B5B0f"
+   },
+   "source": [
+    "남자 승객이 여자 승객보다 대략 2배 많습니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "b03dVV9q5Dv2"
+   },
+   "outputs": [],
+   "source": [
+    "dftrain.sex.value_counts().plot(kind='barh')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "rK6WQ29q5Jf5"
+   },
+   "source": [
+    "대부분의 승객은 \"삼등석\" 입니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "dgpJVeCq5Fgd"
+   },
+   "outputs": [],
+   "source": [
+    "dftrain['class'].value_counts().plot(kind='barh')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "FXJhGGL85TLp"
+   },
+   "source": [
+    "여자는 남자보다 살아남을 확률이 훨씬 높습니다. 이는 모델에서 명확하게 예측가능한 특성입니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "lSZYa7c45Ttt"
+   },
+   "outputs": [],
+   "source": [
+    "pd.concat([dftrain, y_train], axis=1).groupby('sex').survived.mean().plot(kind='barh').set_xlabel('% survive')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "VqDKQLZn8L-B"
+   },
+   "source": [
+    "## 모델을 위한 특성 공학(feature engineering)\n",
+    "추정기는 [특성 열(feature columns)](https://www.tensorflow.org/guide/feature_columns)이라는 시스템을 사용하여 모델이 각각의 입력 특성을 어떻게 해석할지 설명합니다. 추정기가 숫자 입력 벡터를 요구하면, *특성 열*은 모델이 어떻게 각 특성을 변환해야하는지 설명합니다.\n",
+    "\n",
+    "효과적인 모델 학습에서는 적절한 특성 열을 고르고 다듬는 것이 키포인트 입니다. 하나의 특성 열이 원래의 특성 `dict` (*기준 특성 열*)의 원래 입력 중 하나가 되거나 하나 또는 여러 기준열 (*얻어진 특성 열*) 에 정의된 변환을 이용하여 생성된 새로운 열이 될 수 있습니다.\n",
```

Comment:
"A feature column can be either one of the raw inputs in the original features dict" can be misunderstood easily.
You can read "A feature column can be **made by** the raw inputs in the original features dict".
I just want to remove duplicate '원본' and write '딕셔너리' with dict.
I make it short because this sentense is quite a long.
I suggest '하나의 특성 열은 특성 딕셔너리(dict)의 원본 입력으로 만들어진 열(기본 특성 열)이거나 하나 이상의 기본 열(얻어진 특성 열)에 정의된 변환을 이용하여 새로 생성된 열입니다.' :)